File: Code coach problems/Hard/C++/Password_Validation.cpp
#include <iostream>
#include <string>
using namespace std;
int main()
{
	string m;
	cin >> m;
	int specChar = 0;
	int num = 0;
	bool l = m.size() >= 7;
	for(int i = 0; i < m.size(); i++)
	{
		if(isdigit(m[i])) 
			num++;
		else if(!isalnum(m[i])) 
			specChar++;
	}
	cout << ((l && num > 1 && specChar > 1) ? "Strong" : "Weak");
}

File: Code coach problems/Hard/Python/Password_Validation.py
m = input()
specChar = 0
num = 0
l = len(m) >= 7
for i in range(len(m)):
	if m[i].isdigit():
		num+=1
	elif not(m[i].isalpha()) and not(m[i].isdigit()):
		specChar+=1
if l and num > 1 and specChar > 1:
	print("Strong")
else:
	print("Weak")
